helm/loki/values.yaml

```diff
@@ -66,1408 +26,84 @@ loki:
-  terminationGracePeriodSeconds: 30
-  # -- Affinity for memcached-index-queries pods. Passed through `tpl` and, thus, to be configured as string
-  # @default -- Hard node and soft zone anti-affinity
-  affinity: |
-    podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchLabels:
-              {{- include "loki.memcachedIndexQueriesSelectorLabels" . | nindent 10 }}
-          topologyKey: kubernetes.io/hostname
-      preferredDuringSchedulingIgnoredDuringExecution:
-        - weight: 100
-          podAffinityTerm:
-            labelSelector:
-              matchLabels:
-                {{- include "loki.memcachedIndexQueriesSelectorLabels" . | nindent 12 }}
-            topologyKey: failure-domain.beta.kubernetes.io/zone
-  # -- Node selector for memcached-index-queries pods
-  nodeSelector: {}
-  # -- Tolerations for memcached-index-queries pods
-  tolerations: []
-
-memcachedIndexWrites:
-  # -- Specifies whether the Memcached index writes cache should be enabled
-  enabled: false
-  # -- Number of replicas for memcached-index-writes
-  replicas: 1
-  # -- The name of the PriorityClass for memcached-index-writes pods
-  priorityClassName: null
-  # -- Labels for memcached-index-writes pods
-  podLabels: {}
-  # -- Annotations for memcached-index-writes pods
-  podAnnotations: {}
-  # -- Labels for memcached-index-writes service
-  serviceLabels: {}
-  # -- Additional CLI args for memcached-index-writes
-  extraArgs:
-    - -I 32m
-  # -- Environment variables to add to memcached-index-writes pods
-  extraEnv: []
-  # -- Environment variables from secrets or configmaps to add to memcached-index-writes pods
-  extraEnvFrom: []
-  # -- Resource requests and limits for memcached-index-writes
-  resources: {}
-  # -- Containers to add to the memcached-index-writes pods
-  extraContainers: []
-  # -- Grace period to allow memcached-index-writes to shutdown before it is killed
-  terminationGracePeriodSeconds: 30
-  # -- Affinity for memcached-index-writes pods. Passed through `tpl` and, thus, to be configured as string
-  # @default -- Hard node and soft zone anti-affinity
-  affinity: |
-    podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchLabels:
-              {{- include "loki.memcachedIndexWritesSelectorLabels" . | nindent 10 }}
-          topologyKey: kubernetes.io/hostname
-      preferredDuringSchedulingIgnoredDuringExecution:
-        - weight: 100
-          podAffinityTerm:
-            labelSelector:
-              matchLabels:
-                {{- include "loki.memcachedIndexWritesSelectorLabels" . | nindent 12 }}
-            topologyKey: failure-domain.beta.kubernetes.io/zone
-  # -- Node selector for memcached-index-writes pods
-  nodeSelector: {}
-  # -- Tolerations for memcached-index-writes pods
-  tolerations: []
+loki-upstream:
```

Comment:
Not sure I like it being called loki-upstream

Reply:
I encountered some problems with prometheus-operator release being named as `upstream-xxxxx` 
So, I renamed the alias to our old chart name: https://github.com/giantswarm/prometheus-operator-app/blob/upgrade-32.4/helm/prometheus-operator-app/Chart.yaml#L18
and it does the job.

Reply:
I've encountered problems I expected to fix with this name, but I guess they're not valid anymore and were just PEBKACs.
Also I wanted to make sure not to mix names (loki-our-chart vs loki-the-upstream-chart), but I guess now it works I can name it back to its original name, `loki`.

Reply:
PEBKAC? What does it mean?

Reply:
Problem Exists Between Keyboard And Chair :point_right: me :point_left:  :rofl: